pkg/ovsdb/monitor.go

```diff
@@ -588,12 +621,19 @@ func (u *updater) prepareRow(value []byte) (map[string]interface{}, string, erro
 	if err != nil {
 		return nil, "", err
 	}
+	res, err := u.isRowAppearOnWhere(data)
```

Comment:
instead of changing the `getAdnDeleteUUID`, I would put the `isRowAppearonWhere` call in `prepareRow, before calling `getAdnDeleteUUID`

Reply:
I'm calling `isRowAppearonWhere` on `prepareRow` already.
notice that `getAdnDeleteUUID` remains untouched.